src/rastervision/builders/label_store_builder.py

```diff
@@ -31,12 +31,13 @@ def build(config,
             readable=readable,
             writable=writable)
     elif label_store_type == 'segmentation_raster_file':
-        raster_class_map = config.segmentation_raster_file.raster_class_map
-        return SegmentationRasterFile(
-            src=config.segmentation_raster_file.src,
-            dst=config.segmentation_raster_file.dst,
-            raster_class_map=raster_class_map)
-        return None
+        srf = config.segmentation_raster_file
+        if not str(srf.source) == '':
+            return SegmentationInputRasterFile(
+                source=srf.source, raster_class_map=srf.raster_class_map)
```

Comment:
Now that there are separate classes, it might make sense to just replace `source` with `uri` and `sink` with `uri` since it's already clear what they are.

Reply:
:+1: I will do so in a subsequent pull request

Reply:
Actually on second look, there are places where these different attribute names are relied-on.  This this area is the subject of the larger-scale refactor that is going on, it might not make sense to spend too much effort on this.